jobs/JGDAS_ATMOS_VERFRAD

```diff
@@ -77,30 +81,30 @@ export p_cyc=$(echo ${cdate} | cut -c9-10)
 # TANKverf - WHERE OUTPUT DATA WILL RESIDE
 #############################################
 export TANKverf=${TANKverf:-$(compath.py ${envir}/${NET}/${gfs_ver})}
-export TANKverf_rad=${TANKverf_rad:-${TANKverf}/${RUN}.${PDY}/${cyc}/$COMPONENT/radmon}
-export TANKverf_radM1=${TANKverf_radM1:-${TANKverf}/${RUN}.${P_PDY}/${p_cyc}/$COMPONENT/radmon}
+export TANKverf_rad=${TANKverf_rad:-${TANKverf}/${RUN}.${PDY}/${cyc}/${COMPONENT}/radmon}
+export TANKverf_radM1=${TANKverf_radM1:-${TANKverf}/${RUN}.${P_PDY}/${p_cyc}/${COMPONENT}/radmon}
 export COM_IN=${COM_IN:-$(compath.py ${envir}/${NET}/${gfs_ver})}
-export COMIN=${COMIN:-${COM_IN}/${RUN}.${PDY}/${cyc}/$COMPONENT}
+export COMIN=${COMIN:-${COM_IN}/${RUN}.${PDY}/${cyc}/${COMPONENT}}
 
 ################################
 #  backwards compatibility for
-#  gfs v15 which doesn't have 
-#  a $COMPONENT in output path 
+#  gfs v15 which doesn't have
+#  a $COMPONENT in output path
 ################################
 if [[ ! -d ${COMIN} ]]; then
    export COMIN=${COM_IN}/${RUN}.${PDY}/${cyc}
 fi
```

Comment:
Retire this?

Reply:
I can't tell.
Definitely not part of this PR.